core/modules/tests/testApp.py

```diff
@@ -28,9 +28,11 @@
 
 
 class AppTest(unittest.TestCase):
+
```

Comment:
Did `autopep8` add this line? I'd be surprised given that we ran most of the pipelines codebase through `autopep8` and didn't end up with blank lines where there should be no blank lines.